iot-hub/Samples/device/MessageReceiveSample/MessageReceiveSample.cs

```diff
@@ -71,8 +71,6 @@ private async Task ReceiveC2dMessagesPollingAndComplete(TimeSpan timeout)
 
                 await _deviceClient.CompleteAsync(receivedMessage);
                 Console.WriteLine($"{DateTime.Now}> Completed C2D message with Id={receivedMessage.MessageId}.");
-
-                receivedMessage.Dispose();
```

Comment:
initialized using `using Message receivedMessage = await _deviceClient.ReceiveAsync(timeout);`, so will get disposed automatically once out-of-scope